src/witan/gwyn/gwyn.clj

```diff
@@ -67,10 +70,26 @@
 
 (defn collect-property [json-entry]
   (hash-map :address (get json-entry "vicinity")
-            :type (get json-entry "types")
+            :type (set (get json-entry "types"))
             :id (get json-entry "id")
             :name (get json-entry "name")))
 
+(def unwanted-properties
+  (edn/read-string
+   (slurp (io/file "data/unwanted_property_types.edn"))))
```

Comment:
Why a file and not a resource, an input or just data in code? When JAR'd, models won't have access to external files